Armadillo/src/main/java/com/scribd/armadillo/playback/mediasource/DashMediaSourceGenerator.kt

```diff
@@ -46,27 +45,32 @@ internal class DashMediaSourceGenerator @Inject constructor(
         )
 
         return if (isDownloaded) {
-            val drmManager = drmSessionManagerProvider.get(mediaItem)
-            if(request.drmInfo?.drmType == DrmType.WIDEVINE) {
+            val drmManager = if (request.drmInfo == null) {
+                drmSessionManagerProvider.get(mediaItem)
+            } else null
```

Comment:
I wanted to double check that this is right - that if drmInfo is null, then we get a drmManager, but if it's not null then we set drmManager to null? From the desc, I would assume the opposite.